Source secret dialog

debsmita1 opened the pull request:
This PR includes:
- From the Source Secret Dropdown user can select from the already created secrets or can choose to create a new one
- On selecting `Create New Secret` a dialog opens which contains the source secret creation form.
- On completion of the form, when clicks on `Create` the dialog closes and the newly created secret is auto selected in the Source Secret Dropdown
- If user clicks on `Cancel`/`Clicks outside the modal`, the modal closes and Source Secret Dropdown gets back to showing `Select Secret Name`

![2019-06-19-11_26_56 331372](https://user-images.githubusercontent.com/22490998/59740361-828b8c80-9285-11e9-83c3-551ed3c9675f.gif)


Refer Story: https://jira.coreos.com/browse/ODC-1024

rhamilto committed:
Add ConsoleNotification CRD

rhamilto committed:
Add ConsoleLink CRD to enable custom links

rhamilto committed:
Add ConsoleCLITool CRD extension

alecmerdler committed:
create operand form for OLM

sg00dwin committed:
Convert dropdowns and related inputs to pf4

rawagner committed:
Add details card.
Connect dashboard cards to K8s resources with Firehose

mareklibra committed:
Introduce VirtualMachine Details page

TheRealJon committed:
Migrate Gauge component from plotly to PF4

vojtechszocs committed:
Add plugin integration tests

yaacov committed:
Use page route instead of resource for vmtemplates

jhadvig committed:
Add ExpandableAlert component

Commit:
Add Host Status to hosts listing

* Introduces BaremetalHostStatus component which renders appropriate
  status component based on host status
* Introduces utility functions to determine host status (optionally also
  from host's machine and node)

cloudbehl committed:
Initial PR for ceph plugin

christianvogt committed:
add action item feature to dropdown

christianvogt committed:
add knative icon

christianvogt committed:
update eslint rules

christianvogt committed:
add perspective header to main nav

christianvogt committed:
support redirect to perspective landing page

christianvogt committed:
update vscode settings for tsdk and debug.node.autoAttach

christianvogt committed:
update types for page extensions

christianvogt committed:
change Administrator perspective icon to PF CogsIcon

christianvogt committed:
support children in namespace bar

christianvogt committed:
support namespace redirect for any URL that supports namespaces

christianvogt committed:
add active application redux state, action, reducer

christianvogt committed:
update ownerReference types

christianvogt committed:
export routes#getRouteWebURL

christianvogt committed:
export create-secret#SSHAuthSubform

christianvogt committed:
add dev-console dependencies

christianvogt committed:
contribute dev-console plugin

benjaminapetersen committed:
Move cmd/bridge/config.go into pkg

- Moves server config into /pkg/serverconfig instead of in /cmd/bridge
- Splits config into 2 files:
  - types.go is the struct, which is shared with the operator
  - config.go is the helper funcs used by console only, not needed by
the operator
- Moves a few validation funcs into /pkg/bridge

rhamilto committed:
Add standard `Resource > Resource Details` breadcrumb to resource details pages

Fixes https://jira.coreos.com/browse/CONSOLE-1539

TheRealJon committed:
Improve area and bar chart readability

christianvogt committed:
support highlighting active root nav items outside of a section

kyoto committed:
Query Browser: Add available metrics dropdown list

Fetch a list of all available metrics from Prometheus and display them
in a dropdown list.

Selecting a metric from the list automatically inserts it at the current
cursor position. To do this, we store the currently focussed input and
the current cursor position / selected text in component state.

Also wrap the query browser with `withFallback`.

mareklibra committed:
VmDetail: redesign to use ResourceSummary

spadgett committed:
Fix broken ClusterServiceVersion instance links

`apiVersion` and `kind` weren't properly injected into the object, and
depending on the URL, the ClusterServiceVersion resource name wasn't found.

suomiy committed:
VirtualMachine details: add Disks tab

- actions: create & remove VM disks

openshift-merge-robot committed:
Merge pull request #1712 from jhadvig/tag-links

Dont link to ImageStreamTag if there is no tag in ImageStream status

rhamilto committed:
Align <title> with <h1> for "Installed Operators"

TheRealJon committed:
Add polyfill for AbortController

alecmerdler committed:
use metadata.name instead of fieldSelector for OLM e2e cleanup

openshift-merge-robot committed:
Merge pull request #1727 from TheRealJon/bug-1719481

Improve area and bar chart readability

vojtechszocs committed:
Add support for custom table filters

openshift-merge-robot committed:
Merge pull request #1730 from spadgett/broken-csv-isntance-link

Fix broken ClusterServiceVersion instance links

openshift-merge-robot committed:
Merge pull request #1734 from alecmerdler/better-olm-cleanup

Better OLM E2E Cleanup

vojtechszocs committed:
Reuse existing rowFilters prop to contain filter functions

vojtechszocs committed:
Ensure sensible rowFilters default

vojtechszocs committed:
Fix the ordering in getAllTableFilters

openshift-merge-robot committed:
Merge pull request #1710 from kyoto/query-browser-move-hide-button

Query Browser: Make the Hide Graph button also hide the graph controls

openshift-merge-robot committed:
Merge pull request #1733 from rhamilto/csv-title-fix

Align <title> with <h1> for "Installed Operators"

openshift-merge-robot committed:
Merge pull request #1698 from jtomasek/multi-list-page

 Use MultiListPage for BaremetalHostsPage

honza committed:
Add host detail page

openshift-merge-robot committed:
Merge pull request #1701 from kyoto/query-browser-delete-query

Query Browser: Add page Actions menu and kebab menu for each query

rebeccaalpert committed:
frontend: Update alerts to PF4

Updated PF3 alerts and elements with alert-based classes to PF4 alerts.

Fixes https://jira.coreos.com/browse/CONSOLE-1536

kyoto committed:
Query Browser: Allow queries to contain newlines

Change the query input to a <textarea> and allow inserting newlines by
pressing Enter+Shift.

Commit:
Add unit test for limitRange

kyoto committed:
Query Browser: Auto-focus query input

Auto-focus the query input when the page loads and also focus new query
inputs when the "Add Query" button is clicked.

mareklibra committed:
Upgrade kubevirt-web-ui-components to v0.1.35

suomiy committed:
VirtualMachine Detail: add Events tab

rawagner committed:
Add capacity card

rawagner committed:
Add plugin infra for adding new dashboard tabs/cards

jhadvig committed:
Add ImageStream and ImageStream history timeline

openshift-merge-robot committed:
Merge pull request #1735 from vojtechszocs/custom-table-filters

Add support for custom table filters

vojtechszocs committed:
Jest config updates

vojtechszocs committed:
Jest config updates

vojtechszocs committed:
Fix error in Table stateToProps

openshift-merge-robot committed:
Merge pull request #1725 from rhamilto/breadcrumb-changes

Add standard `Resource > Resource Details` breadcrumb to resource det…

spadgett committed:
Remove misleading route hostname message

The create route form says a route hostname can't be changed. This is
not always the case and depends on the user's permissions. Remove the
message.

rhamilto committed:
Fix bug where error occurred when clicking Update Channel Save without interacting with channel dropdown

Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1721431

alecmerdler committed:
improve feature detection for OLM nav items on upstream k8s

spadgett committed:
Make cluster status default page for native k8s

spadgett committed:
Limit alert height in modal editors

openshift-merge-robot committed:
Merge pull request #1738 from mareklibra/upgradeComponents.v0.1.35

Upgrade kubevirt-web-ui-components to v0.1.35

openshift-merge-robot committed:
Merge pull request #1743 from vojtechszocs/jest-config-updates

Jest config updates

openshift-merge-robot committed:
Merge pull request #1744 from vojtechszocs/fix-filter

Fix error in Table stateToProps

openshift-merge-robot committed:
Merge pull request #1746 from rhamilto/bz-1721431

Bug 1721431:  Fix bug where error occurred when clicking Update Channel Save withou…

openshift-merge-robot committed:
Merge pull request #1747 from alecmerdler/CONSOLE-1544

Improve Feature Detection for OLM Nav Items

openshift-merge-robot committed:
Merge pull request #1745 from spadgett/fix-hostname-message

Remove misleading route hostname message

openshift-merge-robot committed:
Merge pull request #1749 from spadgett/modal-alert-height

Limit alert height in modal editors

openshift-merge-robot committed:
Merge pull request #1696 from rawagner/dashboard-k8s-simple

Add details card

openshift-merge-robot committed:
Merge pull request #1748 from spadgett/native-k8s-default-page

Make cluster status default page for native k8s

kyoto committed:
Query Browser: Add switch for toggling queries ON / OFF

kyoto committed:
Query Browser: Monitoring UI PatternFly tables fixes

Fix Alerts table sort by name.
Fix Silences table `aria-label`.

openshift-merge-robot committed:
Merge pull request #1737 from xiaocwan/limitrangeUnitTest

Add unit test for limitRange

kyoto committed:
Query Browser: Avoid unnecessary graph component renders

Memoize `Graph` and `SpanControls` components and also memoize the
`onSpanChange()` and `onZoom()` callbacks.

Move the default `domain` creation into `Graph` so that the `domain`
prop doesn't depend on the current time.

mareklibra committed:
Refactor: split VMDetails to separate file

kyoto committed:
Query Browser: Allow disabling individual series in the graph

It was already possible to disable / enable each PromQL query
individual, but this adds the ability to disable / enable an individual
series (line on the graph) returned by a query.

openshift-merge-robot committed:
Merge pull request #1708 from vojtechszocs/plugin-tests

Add plugin extension tests

divyanshiGupta commented on frontend/public/components/secrets/create-secret.tsx:
```diff
@@ -189,12 +190,45 @@ const withSecretForm = (SubForm) => class SecretFormComponent extends React.Comp
+            <button type="button" className="btn btn-default" id="cancel" onClick={onCancel}>Cancel</button>
+          </ButtonBar>
+        </form>
+      </div>);
```
@christianvogt should we make changes to this file as it is a part of console code? I think we can create a similar file in our codebase that way we can use formik fields(wraps pf3 form components) to build this form.

divyanshiGupta commented on frontend/public/extend/devconsole/components/import/git/SourceSecretSelector.tsx:
```diff
@@ -1,47 +1,91 @@
-  sourceSecret,
+interface CreateSourceSecretModalProps {
+  cancel: (e: Event) => void;
+  close: () => null;
```
I think it should be void and not null

divyanshiGupta commented on frontend/public/extend/devconsole/components/import/git/SourceSecretSelector.tsx:
```diff
@@ -1,47 +1,91 @@
-const SourceSecretSelector: React.FC<SourceSecretSelectorProps> = ({
-  sourceSecret,
+interface CreateSourceSecretModalProps {
+  cancel: (e: Event) => void;
```
Can you specify the exact event type?

divyanshiGupta commented on frontend/public/extend/devconsole/components/import/git/SourceSecretSelector.tsx:
```diff
@@ -1,47 +1,91 @@
+  const onClose = () => {
+    close();
+    onCancel();
+    return null;
```
Why do you want return null?

debsmita1 replied:
@divyanshiGupta  Because the the higher Order Component `withSecretForm` accepts `onCancel` prop whose return type is declared as `null`.

Commit:
Add unit test for checking resourcequota usage icon

suomiy committed:
K8sResourceKind: conform to API types

suomiy committed:
introduce types to @console/shared

suomiy committed:
TableData: add possibility to pass id to the table cell

suomiy committed:
kubevirt: port VM list to PF4 Table

rhamilto committed:
Bump @patternfly/patternfly and @patternfly/react-core to latest (pre-)release

yaacov committed:
Use plural instead of path (#1728)

* use plural instead of path

* change alert path to plural

rhamilto committed:
Fix bug where there is too much space between link and "- default"

Fixes https://jira.coreos.com/browse/CONSOLE-1535

suomiy committed:
kubevirt: add basic actions to VM list

openshift-merge-robot committed:
Merge pull request #1753 from kyoto/monitoring-fix-react-tables

Query Browser: Monitoring UI PatternFly tables fixes

openshift-merge-robot committed:
Merge pull request #1756 from xiaocwan/crq-unit

Add a unit test for checking resourcequota usage icon

christianvogt committed:
enable dev-console plugin in console-app

openshift-merge-robot committed:
Merge pull request #1736 from kyoto/query-browser-metrics-dropdown

Query Browser: Add available metrics dropdown list

openshift-merge-robot committed:
Merge pull request #1759 from rhamilto/bump-patternfly

Bump @patternfly/patternfly and @patternfly/react-core to latest (pre…

openshift-merge-robot committed:
Merge pull request #1755 from kyoto/query-browser-memoize-graph

Query Browser: Avoid unnecessary graph component renders

cloudbehl committed:
Initial PR for noobaa plugin package

vojtechszocs committed:
Fix errors due to lodash module replacement

openshift-merge-robot committed:
Merge pull request #1721 from christianvogt/dev-console

Dev console

openshift-merge-robot committed:
Merge pull request #1760 from rhamilto/console-1535

Fix bug where there is too much space between resource link and "- Default"

openshift-merge-robot committed:
Merge pull request #1719 from benjaminapetersen/backend/pkg/move-config

Move cmd/bridge/config.go into pkg

divyanshiGupta committed:
Dry-run all resource creations in git import flow

openshift-merge-robot committed:
Merge pull request #1767 from kyoto/query-browser-query-switch

Query Browser: Add switch for toggling queries ON / OFF

openshift-merge-robot committed:
Merge pull request #1682 from mareklibra/kubevirt.vmDetails

Kubevirt VmDetails

Commit:
Add unit test for checking quota columns

sahil143 committed:
Create a reusable Progressive Form Component for Advanced Options

create advanced-options component

refactor getGeneratedText

remove commented code

fix route component props

change <p> tag to React.Fragment

move AdvancedOptions to advanced folder

create a controlled component progressiveList

add scrollIntoView for ProgressiveListItem

add comment

rename Advanced to ProgressiveList

Update frontend/packages/dev-console/src/components/import/GitImportForm.tsx

Co-Authored-By: Christian Vogt <cvogt@redhat.com>

Update frontend/packages/dev-console/src/components/import/GitImportForm.tsx

Co-Authored-By: Christian Vogt <cvogt@redhat.com>

Update frontend/packages/dev-console/src/components/import/advanced/ProgressiveList.scss

Co-Authored-By: Christian Vogt <cvogt@redhat.com>

Update frontend/packages/dev-console/src/components/import/advanced/ProgressiveList.tsx

Co-Authored-By: Christian Vogt <cvogt@redhat.com>

Update frontend/packages/dev-console/src/components/import/advanced/ProgressiveList.tsx

Co-Authored-By: Christian Vogt <cvogt@redhat.com>

Update frontend/packages/dev-console/src/components/import/advanced/ProgressiveList.scss

Co-Authored-By: Christian Vogt <cvogt@redhat.com>

make requested changes

return Jsx element from the get genrated text method, add text prop to Progressive List

add unit tests for  ProgressiveList, ProgressiveListItem, getGeneratedText

move Progressive component to progressive folder

remove divider for advanced options

re arrange imports

make requested changes and fix unit tess

add a filter to only get valid children

change progressive list logic  so the options appear in the order they are clicked

use find  instead of filter, remove React.Children

Commit:
fix(pipeline): Fix pipeline broken flow

divyanshiGupta committed:
Integrate routing advanced option in git import flow

Commit:
feat(pipeline): Make Pipeline/Run yamls read-only

openshift-merge-robot committed:
Merge pull request #1771 from divyanshiGupta/integrate-route

Integrate routing advanced option in git import flow

openshift-merge-robot committed:
Merge pull request #1739 from suomiy/kubevirt.eventsPage

VM Detail: add Events tab

vojtechszocs committed:
Add separate script for plugin stats, simplify webpack config

suomiy committed:
kubevirt: move types to a standalone dir

suomiy committed:
expose asAccessReview functionality

openshift-merge-robot committed:
Merge pull request #1766 from vojtechszocs/fix-lodash-replacement

Fix errors due to Lodash module replacement

openshift-merge-robot committed:
Merge pull request #1769 from kyoto/query-browser-query-multiline

Query Browser: Allow queries to contain newlines

sg00dwin committed:
Prevent long unbroken resource names to cause horizontal scroll.
Add .co-break-word mixin
- add word-break: break-word and min-width: 0 to mixin (for Edge and Firefox)

alecmerdler committed:
use _.attempt() when cleaning up OLM e2e tests

openshift-merge-robot committed:
Merge pull request #1778 from abhinandan13jan/task-status

feat(devconsole - pipeline) Make Pipeline/Run read-only and Fix Pipeline/Run List

rhamilto committed:
Remove extra space above <th> to restore spacing above table to what it was

https://jira.coreos.com/browse/CONSOLE-1530?filter=-1

openshift-merge-robot committed:
Merge pull request #1777 from vojtechszocs/fix-node-require-extensions

Add script for plugin stats & simplify webpack config

openshift-merge-robot committed:
Merge pull request #1781 from alecmerdler/fix-olm-e2e-cleanup

Fix OLM E2E Test Cleanup

openshift-merge-robot committed:
Merge pull request #1709 from cloudbehl/ceph-storage-plugin

Initial PR for ceph plugin package

openshift-merge-robot committed:
Merge pull request #1770 from xiaocwan/quota-unit

Add unit test for checking quota columns

openshift-merge-robot committed:
Merge pull request #1775 from TheRealJon/bug-1720045

Bug 1720045: Add polyfill for AbortController

openshift-merge-robot committed:
Merge pull request #1779 from suomiy/actionsAccessReview

expose build AccessReview functionality

debsmita1 committed:
Create Source Secret dialog for the git import form

invalid-email-address committed:
feat(pipeline) Add task status column to Pipeline/Run List

debsmita1 committed:
selected version gets applied

openshift-merge-robot committed:
Merge pull request #1788 from debsmita1/s2i-version-selection

On Create s2i application form, selected version gets applied

rhamilto committed:
Remove invalid, orphaned CSS partial

kyoto committed:
Query Browser: Remove tooltip from query enable/disable switch

openshift-merge-robot committed:
Merge pull request #1791 from rhamilto/remove-orphaned-css

Remove invalid, orphaned CSS partial

spadgett committed:
Fix aria-label for CSV resource table

rhamilto committed:
Refactor .co-m-pane__body styles so border is correctly added

Fixes https://jira.coreos.com/browse/CONSOLE-1533

priley86 committed:
fix(rowFilters): rowFilters scope fix

openshift-merge-robot committed:
Merge pull request #1796 from priley86/filter-fixes

fix(rowFilters): rowFilters scope fix

jcaianirh committed:
Make ux updates to cluster settings pages

openshift-merge-robot committed:
Merge pull request #1792 from kyoto/query-browser-remove-tooltip

Query Browser: Remove tooltip from query enable/disable switch

openshift-merge-robot committed:
Merge pull request #1795 from rhamilto/console-1533

Refactor .co-m-pane__body styles so border is correctly added

debsmita1 committed:
address review comments

removed CreateSourceSecret.tsx file

openshift-merge-robot committed:
Merge pull request #1741 from honza/host-detail

Add host detail page

kyoto committed:
Query Browser: Add <Query /> component

Clean up code by introducing `Query` component and merging the old
`MetricsList` component into it.

Add "Hide all series" / "Show all series" kebab menu action.

Fix alert details page to only show the single series for the alert (not
all series returned by the alert rule).

Add some type definitions.

kyoto committed:
Query Browser: Fix series table colors to match graph

spadgett committed:
Use `tectonic-console-builder:v17` in Dockerfile

TheRealJon committed:
Switch to WHATWG abort controller polyfill

openshift-merge-robot committed:
Merge pull request #1805 from kyoto/query-browser-toggle-series

Query Browser: Allow disabling individual series in the graph

openshift-merge-robot committed:
Merge pull request #1768 from divyanshiGupta/dry-run

Dry-run all resource creations in git import flow

alecmerdler committed:
remove test files from TypeScript exclude

openshift-merge-robot committed:
Merge pull request #1807 from TheRealJon/abortcontroller-polyfill

Bug 1720045: Switch to WHATWG abort controller polyfill

openshift-merge-robot committed:
Merge pull request #1794 from sg00dwin/modal-break

Prevent long unbroken resource names to cause horizontal scroll on modals that display resource name

openshift-merge-robot committed:
Merge pull request #1699 from jhadvig/ist_list

Add ExpandableAlert component

mareklibra committed:
console-shared: add Filter type

jhadvig committed:
Move alerts related styling

openshift-merge-robot committed:
Merge pull request #1360 from rhamilto/console-links-crd

Add ConsoleLink CRD to enable custom links in help and user dropdowns

alecmerdler committed:
show Subscription tab in ClusterServiceVersion detail view

openshift-merge-robot committed:
Merge pull request #1812 from jhadvig/alerts_style

Move alerts related styling

suomiy committed:
imagestream-utils: fix TS error

openshift-merge-robot committed:
Merge pull request #1797 from alecmerdler/OLM-695

Create Operand Form for OLM

spadgett committed:
Fix display of empty probe on container details page

openshift-merge-robot committed:
Merge pull request #1764 from cloudbehl/noobaa-storage-plugin

Initial PR for NooBaa plugin package

openshift-merge-robot committed:
Merge pull request #1782 from rhamilto/console-1530

Remove extra space above <th> to restore spacing above table to what …

openshift-merge-robot committed:
Merge pull request #1804 from spadgett/dockerfile-builder-v17

Use `tectonic-console-builder:v17` in Dockerfile

openshift-merge-robot committed:
Merge pull request #1808 from alecmerdler/type-check-tests

Type-Check Test Files

spadgett committed:
Use unique reference in redux ID

openshift-merge-robot committed:
Merge pull request #1441 from rhamilto/command-line-tools-page

Add ConsoleCLIDownload CRD extension

openshift-merge-robot committed:
Merge pull request #1757 from suomiy/console.type-fixes

TypeScript Enhancements

openshift-merge-robot committed:
Merge pull request #1814 from spadgett/empty-probe

Fix display of empty probe on container details page

divyanshiGupta committed:
Don't set selectedKey when actionItem is clicked

invincibleJai committed:
fix(knative): adds utils to create knative resources

mareklibra committed:
Upgrade kubevirt-web-ui-components to 0.1.36

openshift-merge-robot committed:
Merge pull request #1758 from suomiy/kubevirt.vmList

Kubevirt: port VM list to PF4 Table

suomiy committed:
fix KebabOption type

kyoto committed:
Query Browser: Add a few more type definitions

openshift-merge-robot committed:
Merge pull request #1742 from rawagner/dashboards-page

Add plugin infra for adding new dashboard tabs/cards

christianvogt committed:
update dev-console OWNERS

rhamilto committed:
Revert PR-1782 as it introduces a bug where non-sortable col heads have diff spacing

openshift-merge-robot committed:
Merge pull request #1731 from suomiy/kubevirt.disksPage

 VirtualMachine details: add Disks tab

openshift-merge-robot committed:
Merge pull request #1829 from suomiy/fixKebapActionType

fix KebabOption type

rhamilto committed:
Add  accidentally removed when rebasing

rawagner committed:
Remove incorrect style for dashboard overview details card

rawagner committed:
Fix incorrect height of See All

debsmita1 committed:
remove cluster hostname from sidebar

openshift-merge-robot committed:
Merge pull request #1762 from jhadvig/ist_timeline

Add ImageStream and ImageStream history timeline

openshift-merge-robot committed:
Merge pull request #1817 from kyoto/query-browser-toggle-all-series

Query Browser: Add <Query /> component

openshift-merge-robot committed:
Merge pull request #1831 from rhamilto/revert-1782

Revert PR-1782 as it introduces a bug where non-sortable col heads ha…

spadgett committed:
Minor adjustments to alert wording and `co-alert` styles

openshift-merge-robot committed:
Merge pull request #1828 from mareklibra/web-ui-components-0.1.36

Upgrade kubevirt-web-ui-components to 0.1.36

openshift-merge-robot committed:
Merge pull request #1833 from rawagner/details-style

Remove incorrect style for dashboard overview details card

rhamilto committed:
Replace specific suite name with variable

openshift-merge-robot committed:
Merge pull request #1783 from TheRealJon/console-1408

Migrate Gauge component from Plotly to PatternFly 4

openshift-merge-robot committed:
Merge pull request #1798 from jcaianirh/cluster-status-ux

Make ux updates to cluster settings pages

openshift-merge-robot committed:
Merge pull request #1815 from alecmerdler/CONSOLE-1460

Installed Operator Subscription Tab

openshift-merge-robot committed:
Merge pull request #1834 from rawagner/see-all-style

Fix incorrect height of See All

kyoto committed:
Query Browser: Improve series button accessibility

Convert from a `<div>` to a `<button>` and add `aria-label` and `title`
attributes.

openshift-merge-robot committed:
Merge pull request #1837 from kyoto/query-browser-autocomplete-metrics

Query Browser: Add a few more type definitions

openshift-merge-robot committed:
Merge pull request #1835 from rhamilto/integration-debugging

Replace specific suite name with variable

yaacov committed:
use single quat for regular strings

yaacov committed:
typo

openshift-merge-robot committed:
Merge pull request #1763 from rebeccaalpert/alert-conversion

Update alerts to PF4

vikram-raj committed:
Replace NameValueEditor with SelectorInput for label field

openshift-merge-robot committed:
Merge pull request #1673 from yaacov/kubvirt-plugin-vmtemplate

Kubevirt VmTemplateList

yaacov committed:
Move yaml tempalte to models dir

sg00dwin committed:
Add margin to prevent stacked action buttons from being too close
https://jira.coreos.com/browse/CONSOLE-1553

openshift-merge-robot committed:
Merge pull request #1761 from suomiy/kubevirt.vm-actions

Kubevirt: add basic actions to VM list

openshift-merge-robot committed:
Merge pull request #1838 from kyoto/query-browser-series-button

Query Browser: Improve series button accessibility

yaacov committed:
typo

openshift-merge-robot committed:
Merge pull request #1830 from christianvogt/dc-owners

update dev-console OWNERS

openshift-merge-robot committed:
Merge pull request #1832 from rhamilto/consolelink-model-fix

Add  'crd: true' accidentally removed when rebasing

openshift-merge-robot committed:
Merge pull request #1841 from vikram-raj/labelInput

Replace NameValueEditor with SelectorInput for label field

suomiy committed:
kubevirt: add advanced actions to VM list

spadgett committed:
Bug 1718309: bump js-yaml to 3.13.1

sg00dwin committed:
Merge pull request #1845 from sg00dwin/stacked-actions

Add margin to prevent stacked action buttons from being too close

openshift-merge-robot committed:
Merge pull request #1751 from sg00dwin/pf4-dropdown-merge

Convert dropdowns and related inputs to pf4

openshift-merge-robot committed:
Merge pull request #1848 from debsmita1/cluster-hostname

Remove cluster hostname from sidebar

openshift-merge-robot committed:
Merge pull request #1849 from spadgett/js-yaml

Bug 1718309: bump js-yaml to 3.13.1

openshift-merge-robot committed:
Merge pull request #1772 from sahil143/adv-opt-or

Progressive Disclosure of advanced options

kyoto committed:
Query Browser: Fix dropdown styles

Allow `buttonClassName` and `menuClassName` to also be set on
non-autocomplete dropdowns.

rawagner committed:
Use existing Humanize type.
update image-stream-tag.tsx humanize function usage

TheRealJon committed:
Adjust humanize callback for Area and Bar components (#1)

openshift-merge-robot committed:
Merge pull request #1827 from invincibleJai/fix-knative-utils

fix(knative): adds utils to create knative resources

yaacov committed:
rename template files

kyoto committed:
Query Browser: Fix some Monitoring UI styles

Fix Add Query button style and adjust layout to better match the mocks.

kyoto committed:
Query Browser: Expand / collapse button improvements

Extract into a separate component
Change the icon color to grey
Add `aria-label`

openshift-merge-robot committed:
Merge pull request #1857 from kyoto/query-browser-table-toggle-button

Query Browser: Expand / collapse button improvements

openshift-merge-robot committed:
Merge pull request #1793 from spadgett/csv-resources-aria-label

Fix aria-label for CSV resource table

openshift-merge-robot committed:
Merge pull request #1839 from mareklibra/console-shared.filter

console-shared: add Filter type

openshift-merge-robot committed:
Merge pull request #1816 from spadgett/k8s-redux-id

Use unique reference in redux ID

openshift-merge-robot committed:
Merge pull request #1542 from rhamilto/console-notification-crd

Add ConsoleNotification CRD that can appear above, below, or above and below the page

openshift-merge-robot committed:
Merge pull request #1785 from abhinandan13jan/task-status

feat(pipeline) Add task status column to Pipeline/Run List

openshift-merge-robot committed:
Merge pull request #1842 from yaacov/move-yaml-templates

Move yaml tempalte to models dir

openshift-merge-robot committed:
Merge pull request #1855 from kyoto/query-browser-add-query-button

Query Browser style fixes and allow passing buttonClassName and menuClassName to <Dropdown />

yaacov committed:
Kubevirt VMTemplate create YAML

openshift-merge-robot committed:
Merge pull request #1774 from jtomasek/add-host-status-column

Add host status column

yaacov committed:
Add route to cluster vmtemplates for completeness

openshift-merge-robot committed:
Merge pull request #1723 from rawagner/dashboard-capacity-2

Add capacity card

sg00dwin committed:
Add ability to include dropDownClassName="dropdown--full-width" to <Dropdown> component that doesn't include autocomplete.

openshift-merge-robot committed:
Merge pull request #1872 from sg00dwin/add-dropdown-full-width-to-non-autocomplete-dropdowns

Add ability to include dropDownClassName="dropdown--full-width"

openshift-merge-robot committed:
Merge pull request #1847 from suomiy/kubevirt.vmActionsAdvanced

 kubevirt: add advanced actions to VM list

openshift-merge-robot committed:
Merge pull request #1825 from divyanshiGupta/issue#1126

Don't set new state when actionItem is clicked

yaacov committed:
remove cluster route from vmtemplate

openshift-merge-robot committed:
Merge pull request #1865 from yaacov/kubevirt-vm-template-create-yaml

Kubevirt VMTemplate create YAML

debsmita1 committed:
use secret while creating resources
